app/code/community/Technooze/Tproductschedule/Model/Attribute/Backend/Datetime.php

```diff
@@ -101,7 +107,9 @@ public function compareDateToCurrent($date)
             return false;
         }
 
-        $compareDate = Mage::app()->getLocale()->date($date, Varien_Date::DATETIME_INTERNAL_FORMAT);
+        $compareDate = Mage::app()->getLocale()
+            ->date($date, Varien_Date::DATETIME_INTERNAL_FORMAT)
+            ->setTime('23:59:59');
```

Comment:
@dbashyal This is the actual code update to make date inclusive.  I simply set the time to be before midnight.